src/server/api/controllers/vessel.controller.js

```diff
@@ -0,0 +1,118 @@
+"use strict";
+
+const HttpError = require("../lib/utils/http-error");
+const knex = require("../../config/db");
+
+//GET
+const getVessels = async req => {
+  try {
+    return await knex("vessels").select(
+      "vessels.id",
+      "vessels.organization_id",
+      "vessels.mmsi",
+      "vessels.name"
+    );
+  } catch (err) {}
```

Comment:
Please return the error message back to the API, so whoever is consuming the API has a chance of understanding what went wrong.
```
} catch (err) {
  return err.message;
}
```

Reply:
fixed it.